src/main/java/school/lemon/changerequest/java/introduction/hw1/Task5.java

```diff
@@ -1,12 +1,18 @@
 package school.lemon.changerequest.java.introduction.hw1;
 
 
+import static java.lang.Math.*;
+
 public class Task5 {
     public static double calculateA(double x, double y, double z) {
-        return 0;
+        return 2*(Math.sin(x-(Math.PI)/6))*calculateB(z)/(0.5+(Math.sin(y))*(Math.sin(y)));
     }
 
     private static double calculateB(double z) {
-        return 0;
+        return 1+z*z/(3+(z*z)/5);
```

Comment:
also here you've done 2 multiplications of z. create additional variable and put results of z*z there and reuse it.